Orange/widgets/visualize/owruleviewer.py

```diff
@@ -18,8 +20,9 @@ class OWRuleViewer(widget.OWWidget):
     inputs = [("Data", Table, 'set_data'),
               ("Classifier", _RuleClassifier, 'set_classifier')]
 
-    data_output_identifier = "Filtered data"
-    outputs = [(data_output_identifier, Table)]
+    data_output_identifier = "Selected Data"
```

Comment:
I hate this.

First, it should be upper case -- if for nothing else, to match the case of `ANNOTATED_DATA_SIGNAL_NAME`.

Second, it's probably the only widget which defines a constant for the signal name. This could be a good idea, but only if we do it elsewhere too.

It's not your fault (it was here before), and removing it is not a part of this PR. But since you have to rename it anyway, you could perhaps simply get rid of it. Just a suggestion, you don't have to.